docs/manifest.json

```diff
@@ -3040,6 +3040,37 @@
               "title": "Routing",
               "href": "/docs/guides/routing"
             },
+            {
+              "title": "OAuth",
+              "collapse": true,
+              "items": [
+                [
+                  {
+                    "title": "Overview",
```

Comment:
I wonder if "What are OAuth & OIDC" would be a more informative title for this page...